pkg/cmd/cli/cmd/loginoptions.go

```diff
@@ -133,6 +133,9 @@ func (o *LoginOptions) getClientConfig() (*kclient.Config, error) {
 		switch {
 		case o.InsecureTLS:
 			clientConfig.Insecure = true
+			// insecure, clear CA info
```

Comment:
yeah, or users providing both insecure *and* CA flags, but in that case they should be mutually exclusive and validated (not sure if it is)